c/commons-lang/commons-lang_ubi_8.4.sh

```diff
@@ -1,11 +1,12 @@
+#!/bin/bash -e
 # ----------------------------------------------------------------------------
 #
 # Package       : commons-lang
-# Version       : 3.7
+# Version       : 3.7, 3.8.1
 # Source repo   : https://github.com/apache/commons-lang.git
-# Tested on     : UBI: 8.3
+# Tested on     : UBI: 8.4
 # Script License: Apache License, Version 2 or later
-# Maintainer's  : Jotirling Swami <Jotirling.Swami1@ibm.com>
+# Maintainer's  : Apurva Agrawal<Apurva.Agrawal3@ibm.com>
```

Comment:
Please add required fields like, 
# Language
# Travis-check .

Also, keep the other maintainer name as well.